### PR TITLE
Добавлен Класс для работы с бд + методы add get

### PR DESCRIPTION
**Pull request by evgent007**

### CODE REVIEW

File: users.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    birthday = Column(String)
+    chat_id = Column(Integer)
+
+    def __init__(self, name, birthday, chat_id):
+        self.name = name
+        self.birthday = birthday
+        self.chat_id = chat_id
+
+    def __repr__(self):
+        return "<User('%s','%s', '%s')>" % (self.name, self.birthday, self.chat_id)
```
Comment (Glicher-wp) [inline users.py]: No newline at end of file.
Bring quotes to a consistent style. If you use double - use it throughout the code. Same for the single.

File: data_manager.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def add(self, obj):
+        self.session.add(obj)
+        self.session.commit()
+
+    def getById(self, obj, id):
+        for result in self.session.query(obj).filter_by(id=id):
+            return result
+
+
+
```
Comment (Glicher-wp) [inline data_manager.py]: Too much newlines.

File: main.py [hunk 3]
```diff
@@ -110,6 +119,8 @@ async def process_birthday(message, state):
         async with state.proxy() as data:
             data['birthday'] = birthday
             data['congratulations'] = False
+            user = User(data['name'], data['birthday'], message.chat.id)
+            data_manager.add(user)
     await Form.next()
     await message.reply('Введи любое сообщение что бы начать!')
```
Comment (Glicher-wp) [inline main.py:130]: It's might better use another naming for the file couse we have full bot code with same filename. it's an example code, I guess, so you can use 'example.py' naming. Also you can bring code to a script format to demonstrate how it's work.

File: users.py (hunk 1 above)
Comment (evgent007) [reply users.py]: А можно писать на русском!

**Comment by Glicher-wp** [comment]
Latest version has already merged.

**Pull request closed without merging**